tests/files/test_file.py

```diff
@@ -223,3 +223,12 @@ def test_resolve_file_path_pattern_raise_exception(invalid_path, caplog):
         _ = resolve_file_path_pattern(path_pattern=invalid_path)
     expected_error = f"File(s) not found for path/pattern '{invalid_path}'"
     assert expected_error in str(e.value)
+
+
+def test_get_file_list():
+    """Assert that get_file_list handle kwargs correctly"""
+    resp = get_file_list(path="path", conn_id="conn")
+    assert resp.operator.task_id == "get_file_list"
+
+    resp = get_file_list(path="path", conn_id="conn", task_id="test")
+    assert resp.operator.task_id == "test"
```

Comment:
It is probably worth testing the unique name scenario is working as expected as well:
```
resp = get_file_list(path="path", conn_id="conn", task_id="test")]
assert resp.operator.task_id != "test"
```

Reply:
Added a test case for this.